crates/library/std/src/context.fe

```diff
@@ -103,11 +102,8 @@ pub struct Context {
                 gas: evm::gas_remaining(), 
                 addr, 
                 value,
-                input_offset: buf.offset(), 
-                input_len: buf.input_len(),
-                output_offset: buf.offset(), 
-                output_len: buf.output_len()
-            ) == 1
+                buffer:buf
```

Comment:
@saifalkatout do you feel that the identifier should be `buffer` instead of `buf`?

Reply:
No, I think buffer is better since it will be written more neatly, I guess this is an outdated version ?

Reply:
yes, the specific code line we are commenting on is outdated. I see that all identifiers have been changed to `buffer`

if we change all of the parameter names to `buf`, then we can use `buf` everywhere, so we wont have to write `buffer: buf` or `buf: buffer` anywhere. it can all just be `buf`

Reply:
I guess stick with buffer on everything because it is more easily understood, my only reservation would be that it can be also easily understood using buf. So the extra characters would be meaningless.

Reply:
`buf` is the conventional name for buffer instances and I think we should stick with that

a couple examples:
rust: https://docs.rs/bytes/latest/bytes/buf/trait.Buf.html
java: https://docs.oracle.com/javase/8/docs/api/java/nio/Buffer.html